_component_design/choice-tiles.md

```diff
@@ -0,0 +1,224 @@
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Choice tile states"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/inputs-and-controls/choice-tiles/choice-tile-states.png)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+## Interactions
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Single-select interaction
+
+**Default Interaction**: Like with multi-select, when the user first views the choice tile selector, all tiles are presented in the default state unless a preferred choice is preselected.
+
+**After Selection**: To illustrate that only one selection can be made at a time, the remaining tiles change to the subdued state after a selection is made.
+
+**Change selection**: If a user wants to change their selection, the subdued state changes to the hover state on hover.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Group states for Single select interactions"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/inputs-and-controls/choice-tiles/choice-tile-single-select.png)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Multi-select interaction
+
+**Default interaction**: When the user first views the choice tile selector, all tiles are presented in the default state unless a preferred choice is preselected.
+
+**After selection**: After a selection is made, the selected tile changes to the selected state and the other tiles remain in the default state.
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Group states for multi select interactions"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/inputs-and-controls/choice-tiles/choice-tile-multi-select.png)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+
+## Handling errors
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Single-select error
+
+If there is an error on a selection, change that item to errored state and display an error message below it.
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"A selection produced an error"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/inputs-and-controls/choice-tiles/choice-tile-single-error.png)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Group-select error
+
+If there is an error on the entire group, the message should be placed above the group rather than changing the state of each item.
```

Comment:
...place the message above...

Reply:
updated to "If there is an error on the entire group, place the message above the group rather than changing the state of each item."